package.json

```diff
@@ -137,7 +137,7 @@
     "ngx-bootstrap": "3.0.1",
     "ngx-fabric8-wit": "8.2.0",
     "ngx-feature-flag": "1.0.0",
-    "ngx-launcher": "2.0.9",
+    "ngx-launcher": "3.0.11",
```

Comment:
Update to 3.0.12, which removes the dependency editor and the blank booster completely

Reply:
@gastaldi in version 3.0.12 only dependency editor get removed. Blank booster PR is still open https://github.com/fabric8-launcher/ngx-launcher/pull/452

Reply:
Update to 3.0.13 then ;)